lib/hackerone/client/structured_scope.rb

```diff
@@ -3,24 +3,41 @@
 module HackerOne
   module Client
     class StructuredScope
+      include ResourceHelper
+
       DELEGATES = [
         :asset_identifier,
         :asset_type,
         :eligible_for_bounty,
         :eligible_for_submission,
-        :instruction
+        :instruction,
+        :max_severity,
+        :reference
       ]
 
       delegate *DELEGATES, to: :attributes
 
-      def initialize(scope)
+      def initialize(program_id, scope)
```

Comment:
In keeping with the pattern that was set in the Swag object:

https://github.com/github/hackerone-client/blob/cd44f298bcccbafe8a4e77cbc0ff00d6ee8a9d84/lib/hackerone/client/swag.rb#L9

Let's set this method signature to `scope, program = nil`. This will also allow this change to be backwards compatible. 

```suggestion
      def initialize(scope, program = nil)
```

Here's another callsite that accepts program, for example: https://github.com/github/hackerone-client/blob/cd44f298bcccbafe8a4e77cbc0ff00d6ee8a9d84/lib/hackerone/client/incremental/activities.rb#L11-L12